examples/stringsvc4/main.go

```diff
@@ -0,0 +1,147 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"log"
+	"strings"
+	"flag"
+
+	"github.com/go-kit/kit/endpoint"
+	natstransport "github.com/kirooha/kit/transport/nats"
```

Comment:
github.com/kirooha/kit/transport/nats - maybe need to change to github.com/go-kit/kit/transport/nats